pkg/util/validation/limits.go

```diff
@@ -39,6 +39,7 @@ type Limits struct {
 	RejectOldSamplesMaxAge time.Duration       `yaml:"reject_old_samples_max_age"`
 	CreationGracePeriod    time.Duration       `yaml:"creation_grace_period"`
 	EnforceMetricName      bool                `yaml:"enforce_metric_name"`
+	SubringSize            int                 `yaml:"subring_size"`
```

Comment:
I would call it `user_subring_size` like the CLI flag.

Reply:
Done.